src/pages/resources.js

```diff
@@ -61,6 +63,9 @@ export default class Resources extends Component {
 
 		if (value !== "Show All") {
 			resources = await resources.filter(resource => resource.category.toLowerCase() === value);
+			this.props.router.push(`/resources?category=${value}`);
```

Comment:
As it currently stands this would cause in essence a page refresh - this means any data fetching methods will be ran. This is not really what we want, as we are adding the parameter to the URL as a convenience to the user, not to change data.

Instead i suggest we use shallow routing: https://nextjs.org/docs/routing/shallow-routing

This is essentially the same as you have now, however without the data loading functions running.

E.g
```js
...
    this.props.router.push(`/resources?category=${value}`, undefined, {shallow: true});
} else {
    this.props.router.push("/resources", undefined, {shallow: true});
...
```